stubs/config.php

```diff
@@ -19,6 +19,23 @@
 
     'preset' => 'default',
 
+
+    /*
+    |--------------------------------------------------------------------------
+    | IDE
+    |--------------------------------------------------------------------------
+    |
+    | This options allow to add hyperlinks in your terminal to quickly open
```

Comment:
Can you make this comments just like in Laravel? Every paragraph is 3 chars shorter then the last one.

Reply:
:roll_eyes:  Waw ! I didn't pay attention to that ! It's quite hard :sweat_smile: 

Is it better now ?